utils_pytorch.py

```diff
@@ -103,33 +105,41 @@ def export_model(model, path=None, input_shape=(1, 3, 64, 64)):
     """
     path = get_model_path() if path is None else path
     model = deepcopy(model).cpu().eval()
-    if not isinstance(model, torch.jit.ScriptModule):
-        assert input_shape is not None, "`input_shape` must be provided since model is not a " \
-                                        "`ScriptModule`."
-        traced_model = trace(model, torch.zeros(*input_shape))
-    else:
-        traced_model = model
-    torch.jit.save(traced_model, path)
-    return path
-
-
-def import_model(path=None):
+    if use_script_module:
+        if not isinstance(model, torch.jit.ScriptModule):
+            assert input_shape is not None, "`input_shape` must be provided since model is not a " \
+                                            "`ScriptModule`."
+            traced_model = trace(model, torch.zeros(*input_shape))
+        else:
+            traced_model = model
+        torch.jit.save(traced_model, path)
+        return path
+    else:
+        torch.save(model, path) # saves model as a nn.Module
+        return path        
+
+
+def import_model(path=None, use_script_module=False):
```

Comment:
This needs some work. The argument `use_script_module` can't be set by the user, since `import_model` is for internal use (see below).